mev_inspect/aave_liquidations.py

```diff
@@ -62,17 +64,21 @@ def get_aave_liquidations(
                     debt_purchase_amount=trace.inputs["_purchaseAmount"],
                     protocol=Protocol.aave,
                     received_amount=received_amount,
+                    received_token_address=received_token_address,
                     transaction_hash=trace.transaction_hash,
                     trace_address=trace.trace_address,
                     block_number=trace.block_number,
                 )
             )
+
     return liquidations
 
 
 def _get_liquidator_payback(
```

Comment:
the goal is always that someone can understand a function purely by its signature (name, inputs, and types)

It's not immediately clear what `Tuple[int, str]` is without reading the code

One option is to give it a name (ex: `_get_liquidator_payment_token_address_and_amount`)

Another option is to do what I think you said on the phone and actually return the full transfer (feels cleaner to me than this long name)

Reply:
returning the whole transfer will cause a variable return type. Rather have a better name and constant return type than variable return types